packages/block-editor/src/components/default-block-appender/style.scss

```diff
@@ -53,11 +53,12 @@
 }
 
 // Empty / default block side inserter.
-.block-editor-block-list__empty-block-inserter, // Empty paragraph
+.block-editor-block-list__empty-block-inserter.block-editor-block-list__empty-block-inserter, // Empty paragraph, needs specificity to override inherited popover styles.
 .block-editor-default-block-appender .block-editor-inserter { // Empty appender
 	position: absolute;
 	top: 0;
 
+	height: $icon-button-size-small + $grid-size;
```

Comment:
This (#20633) could use some close attention I think, since the `height` is maybe specific to the new G2 styling? Not sure if this matches what exists in `wp/trunk` (or if it's even necessary? unclear to me how this issue suddenly surfaced)

Reply:
Something is needed here, since I can reproduce the original issue when the `height` is removed. Per #20663, I believe it's a consequence of how block inserters were moved into positioned popovers (probably #19406?). What I'm not so sure about is the specific value, and whether the one introduced with #20663 was targeted at the updated style for the inserter. I do seem to find that `height: auto;` can work okay, or at least that the `height: auto;` is a different effective result than what is currently produced here (28px with `height: auto`, or same as `$block-side-ui-width` from style below, vs. 32px with this `$icon-button-size-small + $grid-size`).

Reply:
Thank you for raising this. The PR was updated to use $block-side-ui-width.